src/Npgsql.CrateDb/CrateDbDatabaseInfo.cs

```diff
@@ -71,7 +71,16 @@ public CrateDbDatabaseInfo(NpgsqlConnection conn)
             {
                 Version = v;
             }
-            
+
+            //Added a type mapping from 'geo_point' (oid 600) to 'NpgsqlPoint', to be
+            //able to handle 'geo_point' values with CrateDB >= 4.1.There the streaming
+            //changed, so that 'geo_point' values are no longer streamed as double array
+            //but as 'geo_point'.
+            if (Version >= new Version("4.1.0"))
+            {
+                CrateDbBaseTypes.Add("point", 600);
```

Comment:
Isn't it possible to always add this, even for CrateDB <= 4.0? 

The server communicates the oid's to the client. in CrateDB 4.0 or earlier it will simply never use `600`, but `1022` (double array)

Reply:
yeah you are right. i will add it to the base mapping. Thx for the info. 
Commit will follow in few minutes